src/subcommand.rs

```diff
@@ -216,7 +216,7 @@ impl Subcommand {
     let chooser = chooser
       .map(OsString::from)
       .or_else(|| env::var_os(config::CHOOSER_ENVIRONMENT_KEY))
-      .unwrap_or_else(|| OsString::from(config::CHOOSER_DEFAULT));
+      .unwrap_or_else(|| config::chooser_default(justfile));
```

Comment:
I think it's more natural to pass in `Search::justfile` here:

```suggestion
      .unwrap_or_else(|| config::chooser_default(&search.justfile));
```

Reply:
Thanks, updated.